docs/developers/how-to/building.rst

```diff
@@ -0,0 +1,81 @@
+==========================
+Build MicroOVN from source
```

Comment:
should we say "Build and install.." or does "build" imply "install" for a developer?

Reply:
That's a good point, they are two distinct steps and one may wish to only build, but not necessarily install.